docs/dynamic_batch_size.md

```diff
@@ -0,0 +1,195 @@
+
+## Steps
+Clone OpenVINO&trade; Model Server github repository and enter `model_server` directory.
+#### Download the pretrained model
+Download model files and store it in `models` directory
+```Bash
+mkdir -p models/resnet/1
+curl https://download.01.org/opencv/2021/openvinotoolkit/2021.1/open_model_zoo/models_bin/1/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin https://download.01.org/opencv/2021/openvinotoolkit/2021.1/open_model_zoo/models_bin/1/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml -o models/resnet/1/resnet50-binary-0001.bin -o models/resnet/1/resnet50-binary-0001.xml
+```
+
+#### Pull the latest OVMS image from dockerhub
+Pull the latest version of OpenVINO&trade; Model Server from Dockerhub :
+```Bash
+docker pull openvino/model_server:latest
+```
+
+#### OVMS configuration file
+Create new file named `config.json` :
+```json
+{
+   "model_config_list": [
+       {
+           "config": {
+               "name": "resnet",
+               "base_path": "/models/resnet"
+           }
+       }
+   ],
+   "pipeline_config_list": [
+       {
+           "name": "resnet50DAG",
+           "inputs": [
+               "0"
+           ],
+           "demultiply_count" : 0,
+           "nodes": [
+               {
+                   "name": "resnetNode",
+                   "model_name": "resnet",
+                   "type": "DL model",
+                   "inputs": [
+                       {
+                           "0": {
+                               "node_name": "request",
+                               "data_item": "0"
+                           }
+                       }
+                   ],
+                   "outputs": [
+                       {
+                           "data_item": "1463",
+                           "alias": "1463"
+                       }
+                   ]
+               }
+           ],
+           "outputs": [
+               {"1463": {
+                       "node_name": "resnetNode",
+                       "data_item": "1463"}}
+           ]
+       }
+   ]
+}
+```
+
+#### Start ovms docker container with downloaded model
+Start ovms container with image pulled in previous step and mount `models` directory :
+```Bash
+docker run --rm -d -v $(pwd)/models:/models -v $(pwd)/config.json:/config.json -p 9000:9000 openvino/model_server:latest --config_path config.json --port 9000
+```
+
+#### Run the client
+```Bash
+cd example_client
+virtualenv .venv
+. .venv/bin/activate
+pip install -r client_requirements.txt
+
+python grpc_serving_client.py --images_numpy_path imgs.npy --labels_numpy_path lbs.npy --input_name 0 --output_name 1463 --model_name resnet50DAG --dag-batch-size-auto --transpose_input False --batchsize 1 ; python grpc_serving_client.py --images_numpy_path imgs.npy --labels_numpy_path lbs.npy --input_name 0 --output_name 1463 --model_name resnet50DAG --dag-batch-size-auto --transpose_input False --batchsize 8;
```

Comment:
It might be easier to read when the output is directly after each command.

Reply:
This is done in such a way because I intended to show that there is no need to reload the config. Should I split the commands as You are suggesting?

Reply:
It might be better to start the clients in background to ensure they run simultaneously with & sign.
The output should be clearly explained and separated to show which part is for which BS.